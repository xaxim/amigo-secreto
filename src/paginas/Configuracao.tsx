import Card from "../componentes/Card"
import Formulario from "../componentes/Formulario"
import ListaParticipantes from "../componentes/ListaParticipantes"
import Rodape from "../componentes/Rodape"

const Configuracao = () => {
    return (
        <Card>
            <section>
                <h2>Vamos come√ßar!</h2>
                <Formulario />
                <ListaParticipantes />
                <Rodape />
            </section>
        </Card>
    )
}

export default Configuracao