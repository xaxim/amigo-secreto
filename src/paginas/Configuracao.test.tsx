import { render } from "@testing-library/react";
import React from "react";
import { RecoilRoot } from "recoil";
import Configuracao from "./Configuracao";

const mockNavegacao = jest.fn()

jest.mock('react-router-dom', () => {
    return {
        useNavigate: () => mockNavegacao
    }
})

describe('a pagina de configuracao', () => {
    test('deve ser renderizada corretamente', () => {
        const { container } = render(<RecoilRoot>
            <Configuracao />
        </RecoilRoot>)

        expect(container).toMatchSnapshot()
    })
})