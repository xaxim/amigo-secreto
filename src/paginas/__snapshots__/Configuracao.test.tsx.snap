// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a pagina de configuracao deve ser renderizada corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section>
      <h2>
        Vamos come√ßar!
      </h2>
      <form>
        <div
          class="grupo-input-btn"
        >
          <input
            placeholder="Insira os nomes dos participantes"
            type="text"
            value=""
          />
          <button
            disabled=""
          >
            Adicionar
          </button>
        </div>
        
      </form>
      <ul />
      <footer
        class="rodape-configuracoes"
      >
        <button
          class="botao"
          disabled=""
        >
          Iniciar brincadeira
        </button>
        <img
          alt="Sacolas de compras"
          src="/imagens/sacolas.png"
        />
      </footer>
    </section>
  </div>
</div>
`;
